Shared/Extensions/JellyfinAPI/MediaSourceInfo+ItemVideoPlayerViewModel.swift

```diff
@@ -14,13 +14,13 @@ import UIKit
 
 extension MediaSourceInfo {
 
-    func videoPlayerViewModel(with item: BaseItemDto, playSessionID: String) throws -> VideoPlayerViewModel {
+    func videoPlayerViewModel(with item: BaseItemDto, playSessionID: String, liveTVChannel: Bool = false) throws -> VideoPlayerViewModel {
 
         let userSession = Container.userSession.callAsFunction()
         let playbackURL: URL
         let streamType: StreamType
 
-        if let transcodingURL, !Defaults[.Experimental.forceDirectPlay] {
+        if let transcodingURL, liveTVChannel ? !Defaults[.Experimental.liveTVForceDirectPlay] : !Defaults[.Experimental.forceDirectPlay] {
```

Comment:
2. LiveTV uses its own Force Direct Play flag, which wasn't being checked

I added a check for the `Defaults[.Experimental.liveTVForceDirectPlay]` flag. In my setup, LiveTV only works with direct play. For some reason the transcoding URL it builds is not able to successfully play back, but I'm not entirely sure why. 

If force direct play is not enabled, `VideoPlayer`'s `onStateUpdated` receives the `.ended` event shortly after opening the video player view. This results in a crash, because the `router` EnvironmentObject seems to not be getting set. The router attempts to dismiss and this crashes.

```
Thread 1: Fatal error: No ObservableObject of type NavigationRouter<VideoPlayerCoordinator> found. A View.environmentObject(_:) for NavigationRouter<VideoPlayerCoordinator> may be missing as an ancestor of this view.
```